charts/traceability-foss/charts/frontend/values.yaml

```
@@ -60,16 +60,23 @@ serviceAccount:
 
 podAnnotations: { }
 
-podSecurityContext: { }
+podSecurityContext:
 # fsGroup: 2000
```

Suggestion: If desired, use this pod configuration instead of container-configuration at line 75.
```suggestion
   fsGroup: 2000
```